app/Libraries/Elasticsearch/Search.php

```diff
@@ -201,9 +228,9 @@ private function fetch()
         try {
             return datadog_timing(
                 function () {
-                    return new SearchResponse(Es::search($this->toArray()));
+                    return new SearchResponse($this->client()->search($this->toArray()));
                 },
-                config('datadog-helper.prefix_web').'.search.fetch',
+                config('datadog-helper.prefix_web').".{$this->statTag}.fetch",
```

Comment:
I'd use the tag array below (`[type' => ...`) instead of changing the metric name. Probably better to add another tag for score type in addition to `type` instead of replacing `type`.